src/pages/viewProfile/index.js

```diff
@@ -0,0 +1,176 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import Button from "../../components/button";
+import "../../pages/viewProfile/viewProfile.css";
+import Card from "../../components/card";
+import ProfileCircle from "../../components/profileCircle";
+import { useParams } from "react-router-dom";
+import { get } from "../../service/apiClient";
+import jwt_decode from "jwt-decode";
+import ErrorMessage from "../../components/errorMessage";
+
+const initialState = {
+  id: "",
+  email: "",
+  role: "",
+  cohort_id: "",
+  firstName: "",
+  lastName: "",
+  biography: "",
+  githubUrl: "",
+  specialism: "",
+};
+// TODO:
+// update the hard-coded start date, end date and job title in the initial state and update it on jsx too. (when server team updates data)
+
+function ViewProfile() {
+  // STATES
+  const [profile, setProfile] = useState(initialState);
+  const [loggedInUserInfo, setLoggedInUserInfo] = useState({});
+  const [isError, setIsError] = useState(false);
+
+  // GLOBAL VARIABLES
+  const navigate = useNavigate();
+  const { id } = useParams();
+  const token = localStorage.getItem("token");
```

Comment:
You can get the current users token from the `useAuth()` hook, e.g. `const { token } = useAuth()`